docs/discover/search-sessions.asciidoc

```diff
@@ -14,10 +14,7 @@ and when your session is complete, view and manage it in *Stack Management*.
 [role="screenshot"]
 image::images/search-session.png[Search Session indicator displaying the current state of the search, which you can click to stop or save a running Search Session ]
```

Comment:
This image needs to be updated.  The search session popup now also includes the date.